apps/src/netsim/NetSimShard.js

```diff
@@ -38,16 +38,25 @@ var NetSimShard = module.exports = function (shardID, pubSubConfig) {
   var channel = this.pubSub.subscribe(shardID);
 
   /** @type {NetSimTable} */
-  this.nodeTable = new NetSimTable(channel, shardID, 'n');
+  this.nodeTable = new NetSimTable(shardID, 'n', {
+    channel: channel
```

Comment:
Put this back the way it was (but did add a truly optional `options` parameter).  Updated the jshint for this file to permit 4-argument method signatures.  I just like the sanity check on making them longer.